Dockerfile

```diff
@@ -19,25 +13,26 @@ RUN apk update \
 ENV GO111MODULE=on
 ENV GOPATH=/
 
-RUN mkdir /opt/uzi-servercertificaat-issuer && cd /opt/uzi-servercertificaat-issuer
+RUN mkdir /opt/go-didx509-toolkit && cd /opt/go-didx509-toolkit
 COPY go.mod .
 COPY go.sum .
 RUN go mod download && go mod verify
 
 COPY . .
-RUN GOOS=$TARGETOS GOARCH=$TARGETARCH go build -ldflags="-w -s " -o /opt/uzi-servercertificaat-issuer/issuer
+RUN GOOS=$TARGETOS GOARCH=$TARGETARCH go build -ldflags="-w -s" -o /opt/go-didx509-toolkit/issuer
 
 # alpine
-FROM alpine:3.20.3
+FROM alpine:3.21.2
 RUN apk update \
   && apk add --no-cache \
              tzdata \
-             curl \
-  && update-ca-certificates
-COPY --from=builder /opt/uzi-servercertificaat-issuer/issuer /usr/bin/issuer
+             curl
```

Comment:
I don't think we need curl, since it's used for healthchecks? (and we have none in the Docker image)